plugin/role_set_test.go

```diff
@@ -0,0 +1,30 @@
+package gcpsecrets
+
+import (
+	"fmt"
+	"testing"
+)
+
+func Test_roleSetServiceAccountDisplayName(t *testing.T) {
```

Comment:
Style suggestion: I think that [table tests](https://dave.cheney.net/2019/05/07/prefer-table-driven-tests) would be cleaner and clearer here.